lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -1193,9 +1123,9 @@ def undelete_workflow(
     )
     def show_versions(
         self,
-        workflow_id: StoredWorkflowIDPathParam,
         trans: ProvidesUserContext = DependsOnTrans,
-        instance: Optional[bool] = InstanceQueryParam,
+        workflow_id: DecodedDatabaseIdField = StoredWorkflowIDPathParam,
```

Comment:
`StoredWorkflowIDPathParam` is a type, `workflow_id: StoredWorkflowIDPathParam,` was the correct way to do this.